CHANGELOG.rst

```diff
@@ -74,6 +74,7 @@ future
 
 - Fix default type of ``JSONField``
 - Install on Windows does not require a C compiler any more.
+- Allow F expression to ``Function``. (#301)
```

Comment:
Could you please expand this a bit. Whilst technically correct, the average user will not see the implication.
Rather say ``Allow usage of F expressions in annotations`` ?

Reply:
Sure, I change changelog and also resolve conflicts :smile: